app/controllers/admin/profiles/admin_profiles_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Admin
+  module Profiles
```

Comment:
Perhaps not one for this pr but we definitely need to standardise these modules / namespaces. Some old designs I've seen had /admin/suppliers and /admin/users as the top spaces and all the different types of users came under these two.

Reply:
Deleted